onnxruntime/core/framework/sequential_executor.cc

```diff
@@ -445,7 +511,7 @@ Status SequentialExecutor::Execute(const SessionState& session_state, const std:
 
   VLOGS(logger, 1) << "Fetching output.";
   // ExecutionFrame::Finalize will update 'fetches' with the final output
-  ORT_RETURN_IF_ERROR(frame.GetOutputs(fetches));
+  ORT_RETURN_IF_ERROR(frame.GetOutputs(fetches, true));
```

Comment:
>true [](start = 48, length = 4)

always release output? Seems like a behavior change for inference.

Reply:
In the past the net effect was the same since after outputs were returned the execution frame was destroyed after this function call and reference was released. I should may be make this as default behavior. 

---
In reply to: [598072449](https://github.com/microsoft/onnxruntime/pull/7077#discussion_r598072449) [](ancestors = 598072449)

Reply:
Is the comment on the previous line still accurate?

Reply:
most certainly not :)

---
In reply to: [599009290](https://github.com/microsoft/onnxruntime/pull/7077#discussion_r599009290) [](ancestors = 599009290)